tokio/src/task/task_local.rs

```diff
@@ -446,3 +447,17 @@ impl From<std::thread::AccessError> for ScopeInnerErr {
         Self::AccessError
     }
 }
+
+/// Returns the `Id` of the currently executing task.
+///
+/// # Panics
+///
+/// This function panics if called from outside a task context
+///
+#[cfg_attr(docsrs, doc(cfg(all(feature = "rt", tokio_unstable))))]
+#[cfg_attr(not(tokio_unstable), allow(unreachable_pub))]
+#[allow(dead_code)]
+pub fn current_id() -> Id {
+    use crate::runtime::context;
+    context::current_task_id()
+}
```

Comment:
Why is this in the `task_local.rs` file? It has nothing to do with task locals as far as I can tell.